apps/civicsignalblog/src/payload/blocks/WebTools/ResourceList.js

```diff
@@ -0,0 +1,24 @@
+const PageHeader = {
+  slug: "webtools-resource-list",
+  labels: {
+    singular: "Resource List",
+    plural: "Resource Lists",
+  },
+  imageURL: "/images/cms/blocks/resource_list.png",
+  imageAltText: "A list of resrouces to display",
```

Comment:
```suggestion
  imageAltText: "A list of resources to display",
```

Reply:
Isn't it better to call this `MediaDataList`?

Reply:
@kilemensi My thinking was to find a generic name, since this could be used to choose if we want to display a list of items could be MediaDataList, or Application Features list etc. in other pages....but looking at the current approach which would use minimal update on the codebase, I think this block will be scrapped of...unless we still want to retain movable/dynamic blocks rendering on the web-tools frontend